readme.md

```diff
@@ -173,7 +176,9 @@ func main() {
     if res.Error != nil {
         log.Fatalf("err: %v", res.Error)
     }
-    fmt.Println(res.Body)
+
+    responseData,_  := res.Body.(*[]ResponseStruct])
```

Comment:
@CeoFred there's an error in this type assertion. I think the token `]` has been unexpectedly added.

Reply:
Fixed @opensaucerer